src/asm_parser.rs

```diff
@@ -593,7 +598,7 @@ exit
     fn test_initial_whitespace() {
         assert_eq!(
             parse(
-                " 
+                "
```

Comment:
Not sure what happened here, I tried bringing the space back but it seems like it gets clipped when I commit (it could be that my editor clipps dangling whitespace) or some linux CRLF / LF issue

Reply:
Ok I used a different editor and managed to get this diff to disappear